docs/ref/contrib/auth.txt

```diff
@@ -628,6 +628,16 @@ The following backends are available in :mod:`django.contrib.auth.backends`:
    users because :attr:`~RemoteUserBackend.user_can_authenticate` always
    returns ``True``.
 
+.. class:: DefaultObjectPermissionsBackend
+
+   .. versionadded:: 2.2
+
+   The default auth backend, :class:`ModelBackend`, unconditionally returns
+   ``False`` if :meth:`~django.contrib.auth.models.User.has_perm` is passed an
+   object. In some cases this is unexpected. Use this backend to make
+   :ref:`object permissions <object_permissions>` a superset of global
+   permissions.
```

Comment:
(Continuing from [mailing list thread](https://groups.google.com/d/topic/django-developers/5vpc_DExhUw/discussion)...) 

Here's one place where it seems under documented.